historical_surveillance/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 3.1.7 on 2021-03-31 18:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AggregateEnrollment',
```

Comment:
@mildredfakoya , is it possible for you to provide us a data script/seed script/migration for getting the data into our local system? I wanted to play around and test the screens you created.

Reply:
Yes. I am just doing a final clean of the data by merging the same school with different names. Almost done now and will send all the cleaned files to you shortly.